README.md

```diff
@@ -189,3 +190,5 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
 This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+,3
```

Comment:
This is a lovely PR!

```suggestion
```

Can I suggest removing these two lines? :) <3

Reply:
Yeah let me take a look tomo